LocalSettings.php

```diff
@@ -4489,6 +4492,14 @@
 	],
 ];
 
+// Email notifications on privileged actions configuration
+'wgMirahezeMagicLogEmailConditions' => [
+	'default' => [
+		'right' => 'watched-person',
```

Comment:
wgMirahezeMagicLogEmailConditions supports multiple watched groups, those values need to be wrapped inside an array